examp/tun.toml

```diff
@@ -96,3 +87,17 @@ abcde = { version = "0.30", features = ["edfg"] }
 [build-dependencies]
 serde_json = "1"
 serde = { version = "1", features = ["derive"] }
+
+[target.'cfg(windows)'.dependencies.windows-sys]
```

Comment:
Can you use the same `cfg` here, i.e. `cfg(target_os = "windows")` to show that it gets sorted directly after the other `windows` dependencies?

Reply:
Or use `cfg(windows)` for the other one, don't really mind.